public/static/locales/en-US/landing.json

```diff
@@ -29,6 +29,27 @@
         "gotIt": "Got it"
       }
     },
+    "patchNotes": {
+      "title": "New in v{{version}}",
```

Comment:
However, I am wondering if the update isn’t the same, like for example you didn’t update these specific things and it were something else. How would you diffidently change the data? Or is this just mock up?

Reply:
I don’t understand? Are you asking how the version number is updated? That’s controlled by the `NEXT_PUBLIC_VERSION` environment variable, which I set on the Vercel (our host) dashboard every time we merge to main.

Can you be more specific?

Reply:
I meant like the data for the update let's say for example, the update was about adding a chinese language to the system. How would you put that into the json file if you used like hard-coded data?

Reply:
You have to change the code to add new features anyway, so I think it makes sense to change more of the source code for the patch notes.

What I’m saying is that it’s completely acceptable to “hard-code” patch notes data, because you already have to touch the code anyway.